### DISCUSSION TITLE
KnowledgeSystem/FrontEnd/HTML/htmlSemanticization

### DISCUSSION THREAD

**Discussion by giscus** [rockbenben/LearnData#48] [Comments]
# KnowledgeSystem/FrontEnd/HTML/htmlSemanticization

如何理解 HTML 语义化 HTML 语义化——这个概念其实诞生了挺长时间，经常发现在面试 JD（Job Description） 中出现要求候选者「了解 HTML 语义化」、「对 HTML 语义化有深刻认知」的需求 对于这么一句 JD 范式标配，如果面试官真的问起，该如何回答呢？ 语义化是什么、为什么、怎么做 简单来说，HTML 语义化就是： 那么为...

https://www.zxwin0125.top/KnowledgeSystem/FrontEnd/HTML/htmlSemanticization.html

**Comment by zxwin0125** [comment]
已完成整理
